### PR TITLE
Update release notes for October 2021 releases

### PR DESCRIPTION
**Pull request by sxa**
List of fixes as per JBS.

Signed-off-by: Stewart X Addison <sxa@redhat.com>

##### Checklist

- [x] `npm test` passes
- [x] documentation is changed or added (if applicable)
- [ ] permission has been obtained to add new logo (if applicable)
- [x] contribution guidelines followed [here](https://github.com/adoptium/adoptium.net/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/handlebars/release_notes.handlebars [hunk 1]
```diff
@@ -19,6 +19,710 @@
       </p>
     </div>
 
+    <div class="anchor">
+      <a href="#jdk8u312" class="anchor"><img src="dist/assets/anchor.png" alt="anchor icon"></a>
+      <h2 class="bold"><button type="button" onclick="showHideReleaseNotes('jdk8u312_notes');">-/+</button> <a id="jdk8u302">OpenJDK 8u312 (1.8.0_312-b07)</a></h2>
```
Comment (karianna) [inline src/handlebars/release_notes.handlebars]: id should be jdk8u312

File: src/handlebars/release_notes.handlebars [hunk 2]
```diff
@@ -19,6 +19,710 @@
...
+    </div>
+    <div class="margin-bottom softHide" id="jdk8u312_notes">
+
+    Here is the list of changes in jdk8u312. For more information on any of these,
+    <a href="https://bugs.openjdk.java.net/browse/JDK-8275549?jql=project%20%3D%20JDK%20AND%20fixVersion%20%3D%20openjdk8u312%20ORDER%20BY%20created%20DESC">
+    look at JBS</a>.
+
+    <ul>
+    <li> JDK-8275549: Unable to read certain PKCS12 keystores from SequenceInputStream</li>
```
Comment (karianna) [inline src/handlebars/release_notes.handlebars]: nitpick - indent the li's for all the sections

Comment (karianna) [reply src/handlebars/release_notes.handlebars]: And indent all of this from the div (for all the sections)

**Review by karianna — CHANGES_REQUESTED** [review]
2 small changes and if you can be bothered converting the `&` to `&amp;` then that might be worth doing as well.

File: src/handlebars/release_notes.handlebars (hunk 2 above)
Comment (sxa) [reply src/handlebars/release_notes.handlebars]: Done.

File: src/handlebars/release_notes.handlebars (hunk 1 above)
Comment (sxa) [reply src/handlebars/release_notes.handlebars]: Good spot. Fixed.

**Comment by sxa** [comment]
> 2 small changes and if you can be bothered converting the `&` to `&amp;` then that might be worth doing as well.

Yep that's a good point and there were a few of them which I've translated. I suspect some of the titles may have got truncated as well but in the grand scheme of things I'm not overly concerned about that :-)

**Review by karianna — APPROVED** [review]

**Pull request merged**